index.html

```diff
@@ -73,6 +73,17 @@ <h2>
         closely matches the end-user's preferences or environment.
       </p>
     </section>
+    <section>
+      <h2>
+        Privacy and security considerations
+      </h2>
+      <p>
+        [=Fetching an image resource=] can reveal the user's IP address. It is
+        RECOMMENDED that user agents fetch the chosen [=image resources=] when
```

Comment:
Fetching anything will reveal the user's IP address, so I'm not sure this is super helpful beyond what's already in fetch. 

Maybe what we can do is talk about any default mitigations we've put in place around requiring CORS, referrer settings, etc.

Reply:
Yes. And this should help with addressing #25 as well.

Reply:
Hey all,

Revealing the IP address is more about preventing tracking, in case a UA periodically refetches the resource. It's probably worth mentioning. I'll reword it in the next commit after we get consensus on the points below.

As for the request defaults, those can be changed by whatever API is using `ImageResource` to suit their use-case. I think those should be brought up in the API's privacy section.

As for CSP, it is applied to the fetch, depending on the `Request` [client](https://fetch.spec.whatwg.org/#concept-request-client) that has been set. See relevant CSP checks within the fetch algorithm here: https://w3c.github.io/webappsec-csp/#should-block-request. Maybe adding a note in this spec's fetch algorithm that APIs using `ImageResource` should always set a `client` would work? Or potentially always enforcing a non-null `client`? WDYT?

Reply:
I don’t know enough about the inner workings of the fetch algorithm to make a call on which wording is better, they both make sense to me. The rest of your recommendations sound good too.

Reply:
> Maybe adding a note in this spec's fetch algorithm that APIs using ImageResource should always set a client would work? 

yes, requiring a client would be good. We can enforce that in the algorithm also, not just note it (i.e., "if client is null, reject/failure" or whatever is appropriate).

Reply:
Cool, I updated the fetch algorithm (and added a note explaining why this is needed for CSP).

I also removed setting the mode to no-cors since that is already the default.

I'll wait for one of you to approve before merging.